src/app/submission/form/collection/submission-form-collection.component.html

```diff
@@ -1,50 +1,44 @@
-<div>
```

Comment:
Please consider there is a [new PR](https://github.com/DSpace/dspace-angular/pull/708) that introduce a refactoring of this component. I should revert these changes because will generate a conflicts unnecessary